### ISSUE TITLE
Fix drive distance PID

### ISSUE THREAD

**Issue by kreeve** [FRC125/NU16#56]
Taha and I identified a few possible bugs during AZ. We should try to fix those so we don't have to rely on a timeout.

**Comment by ghost** [comment]
Yes, I believe we posted them on slack. There seemed to be minor flaws in our logic, but mostly one line fixes?

**Comment by camilo86** [comment]
I'm not entirely sure if this was the error, but the PID `enable()` was commented out along with `setInputRange()` & `setOutputRange()` I added it back in here 4bbaffc00023b6116f3b6872413dc1b6e6572430
